nixos/modules/installer/tools/tools.nix

```diff
@@ -134,6 +134,7 @@ in
       $bootLoaderConfig
         # networking.hostName = "nixos"; # Define your hostname.
         # networking.wireless.enable = true;  # Enables wireless support via wpa_supplicant.
+        # networking.networkmanager.enable = true;  # Easiest to use and most distros use this by default.
```

Comment:
Maybe it could be made more clear that `networking.wireless.enable` and `networking.networkmanager.enable` are mutually exclusive?

Reply:
i added
```
# Pick only one of the below networking options.
```